docs/services/sensors.md

```diff
@@ -29,7 +26,7 @@ Go to your robot's **Code sample** tab on the [Viam app](https://app.viam.com) f
 
 {{% /alert %}}
 
-### Sensors
+### GetSensors
```

Comment:
```suggestion
### Sensors
```
I think this needs to stay `Sensors`, as least as long as it's defined so in the interface here https://pkg.go.dev/go.viam.com/rdk/services/sensors#Service and client file. Same for `Readings`!

Reply:
So, I realize that in the Go SDK/RDK it's Sensors and Readings, but in the proto it's GetSensors and GetReadings. And in the Python it's get_sensors and get_readings. This is the same exact thing that we see across other APIs and we made the decision with those to use what's in the proto.

Reply:
https://github.com/viamrobotics/api/blob/main/service/sensors/v1/sensors.pb.go

Reply:
This is the one thing I would say still blocks-- I really don't think we should change for the Go Client's sake and also for modular resource creaters. Can discuss with everyone but I think it needs to be consistent with https://github.com/viamrobotics/rdk/blob/main/components/base/client.go because that is the actual client and if the method's names are off users will be confused